cpp/tests/io/orc_test.cpp

```diff
@@ -847,4 +856,114 @@ TYPED_TEST(OrcChunkedWriterNumericTypeTest, UnalignedSize2)
   CUDF_TEST_EXPECT_TABLES_EQUAL(*result.tbl, *expected);
 }
 
+TYPED_TEST(OrcReaderTest, SingleColumnSkipRows)
```

Comment:
I think these tests can be restructured to avoid code repetition.
You can make a function that takes the number of rows, skip_rows, and num_rows and performs the testing - you can use skip_rows + now_rows as condition for whether to compare against expected table, or to expect a throw (no need to create the expected table in this case).
Then, all these tests (and more) could be a single test that calls this function with (100, 2, 0), (100, 2, 50), (100, 2, 150), (100, 0, 150). I would also include case (100, 75, 75).